Makefile

```diff
@@ -38,6 +38,14 @@ help: ## Display this help.
 
 ##@ Development
 
+lint-autofix: ## Autofix all possible linting errors.
+	golangci-lint run -E goimports --fix
```

Comment:
Couldn't we reuse the `.golangci.yaml` for fixing? ie instead of `-E`, use the file?

Reply:
In this case, we need to run only one linter, specifically `goimports`, which is able to fix errors related to incorrect formatting of imports. Running all the linter does not seem to be optimal in my opinion. This target in the Makefile is supposed to be used, by default, only to fix incorrectly formatted imports.

Reply:
Couldn't we also fix other lints that do support fixing?
Is this only imports? Probably general formatting could also be done this way, possibly whitespace changes (if we have that lint enabled) etc

Could we do something like "all from config, except non-fast"?  IIRC golangci-lint had some tag for fast lints?

Reply:
For the moment, we just need the goimports fix to be available. Most of the errors that can happen are errors regarding imports. If we see that adding more linters will help solve problems that may arise during further implementation, we will do so in the next iteration. Unfortunately, I could not find in the golangci documentation, a flag that describes a linter as fast.